README.md

```diff
@@ -71,3 +71,15 @@ val message = session.decrypt(Message.PreKey("oun02024f=ocnaowincd;53tnv024ok/7u
 session.clear()
 account.clear()
 ```
+
+#### JS
+
+Javascript has a special prerequisites.
+Before being able to use the matrix-kt lib you need to initiate the olm library.
+
+```js
+require("@matrix-org/olm").init()
```

Comment:
Can this not be done in Kotlin?

Reply:
I came to the conclusion that we should not do that.
This is because init takes some arguments and functions to tell the loader where it might find a the native binary or the wasm file (if run in a browser) and so we would just add a wrapper to the matrix-org olm lib which seems completely unnecessary.

Reply:
Oh I see, alright then.